logging/README.adoc

```diff
@@ -620,7 +620,7 @@ Now you have deployed the quickstart with the default configuration, you run a m
 +
 [source,subs="+quotes,attributes+",options="nowrap"]
 ----
-$ __{jbossHomeName}__/bin/jboss-cli.sh --connect --file=configure-logging.cli
+$ __{jbossHomeName}__/bin/jboss-cli.sh --connect --properties=cli.properties --resolve-parameter-values --file=configure-logging.cli
```

Comment:
Do we need to mention something about OpenShift here and using the `cli-openshift.properties` here instead for that environment?

Reply:
For OpenShift the script is executed when provisioning so all good.